front/src/views/profile/accommodation/accommodation.view.html

```diff
@@ -1,38 +1,74 @@
+        @change="updateQuery" />
+      <v-btn-toggle v-model="sortInfo.sortDirection" mandatory rounded height="40" class="mr-5" dark
+        background-color="accent lighten-1" @change="updateQuery">
+        <v-btn height="40" color="accent lighten-1">
+          <v-icon>mdi-arrow-down</v-icon>
+        </v-btn>
+        <v-btn height="40" color="accent lighten-1">
+          <v-icon>mdi-arrow-up</v-icon>
+        </v-btn>
+      </v-btn-toggle>
+      <v-menu offset-y :close-on-content-click="false" nudge-bottom="20">
+        <template v-slot:activator="{ on, attrs }">
+          <v-btn v-bind="attrs" v-on="on" rounded class="details accent lighten-1">
+            <span class="text-capitalize">
+              {{ $t('accommodations.filter') }}
+              <v-icon right class="white--text">mdi-filter-variant</v-icon>
+            </span>
+          </v-btn>
+        </template>
+        <v-sheet class="px-5 pb-10 pb-3 d-flex flex-wrap">
+          <v-select class="filter-countries mr-5 mt-10" persistent-hint rounded filled dense hide-details multiple
+            :items="countries" append-icon="mdi-chevron-down" color="accent lighten-1"
+            :placeholder="$t('accommodations.country')" v-model="filters.countries" />
+          <v-text-field class="filter-year mr-5 mt-10" filled dense :prefix="$t('accommodations.year') + ': '"
+            hide-details rounded filled single-line dense type="number" color="accent lighten-1"
+            v-model="filters.year" />
+          <v-range-slider :max="maxPrice" :min="0" hide-details class="filter-price mr-5 mt-10" thumb-label="always"
+            color="accent lighten-1" prepend-icon="mdi-currency-usd" v-model="filters.price"
+            @mousedown="filters.isPriceChosen = true" />
+          <v-btn rounded class="details accent lighten-1 mt-10 mr-5" @click="onFilterUpdate">
+            {{ $t('tripHistory.search') }}
+          </v-btn>
+        </v-sheet>
+      </v-menu>
     </div>
   </div>
   <div class="d-flex flex-row align-stretch justify-space-between py-5">
     <div class="align-self-stretch">
-      <div class="d-flex flex-wrap justify-space-between"
-           v-if="accommodations.length">
+      <div class="d-flex flex-wrap justify-space-between" v-if="accommodations.length">
         <template v-for="(accommodation,n) in accommodations">
-          <mmt-accommodation-card :key="`accommodation-${n}`"
-                                  :accommodation="accommodation"
-                                  :class="`${ selectedAccommodation && accommodation.id === selectedAccommodation.id ? 'active' : ''} mb-5`"
-                                  v-on:click.native="viewAccommodation(accommodation, $event)" />
+          <mmt-accommodation-card :key="`accommodation-${n}`" :accommodation="accommodation"
+            :class="`${ selectedAccommodation && accommodation.id === selectedAccommodation.id ? 'active' : ''} mb-5`"
```

Comment:
:class="`${'active' :  selectedAccommodation && accommodation.id === selectedAccommodation.id} mb-5`"

Reply:
I think, this shouldn't work...

Reply:
why? you try it?

Reply:
`'active' : selectedAccommodation && accommodation.id === selectedAccommodation.id`, what do you mean?
Is it a ternary operator or what? I don't understand

Reply:
when the conditions described are met (selectedAccommodation && accommodation.id === selectedAccommodation.id) returns true. from here it follows that another class, ative, is added to the classes, otherwise it will not be added. true or false does not have to be returned, since the condition returns it anyway and it's better reading

Reply:
```javascript
:class="{active: selectedAccommodation && accommodation.id === selectedAccommodation.id}" class="mb-5"
```
this should work